diversity-lunch-be/src/main/java/de/brockhausag/diversitylunchspringboot/profile/model/entities/HobbyCategoryEntity.java

```diff
@@ -0,0 +1,41 @@
+package de.brockhausag.diversitylunchspringboot.profile.model.entities;
+
+import de.brockhausag.diversitylunchspringboot.profile.utils.baseApi.BaseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity
+public class HobbyCategoryEntity implements BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String descriptor;
+
+    @Override
+    public int hashCode() {
+        String hashString = this.id.toString() + this.descriptor;
+        return hashString.hashCode() ;
+    }
```

Comment:
Kann genauso wie bei allen anderen Entities in die BaseEntity

Reply:
Problem ist, dass man dann die BaseEntity abstrakt machen muss und dann durch Getter, Setter und das Id-Attribut noch mehr Code schreiben muss.